lib/authorizer/digest.js

```diff
@@ -91,32 +89,21 @@ module.exports = {
     /**
      * Computes the Digest Authentication header from the given parameters.
      *
-     * @param {Object} params
-     * @param {String} params.algorithm
-     * @param {String} params.username
-     * @param {String} params.realm
-     * @param {String} params.password
-     * @param {String} params.method
-     * @param {String} params.nonce
-     * @param {String} params.nonceCount
-     * @param {String} params.clientNonce
-     * @param {String} params.opaque
-     * @param {String} params.qop
-     * @param {String} params.uri
+     * @param {AuthInterface} auth
      * @returns {String}
      */
-    computeHeader: function (params) {
```

Comment:
This is the only Auth where `computeHeader` takes `AuthInterface` instead of Object. Should we stick to one?